File: INICIANTE/JAVASCRIPT/1134.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) { return lines.shift(); };

var n = prompt("")
var a = 0
var g = 0
var d = 0

for(;n != 4;){
    if(n == 1){
        a = a + 1
    }
    else if(n == 2){
        g = g + 1
    }
    else if(n == 3){
        d = d + 1
    }
    var n = prompt("")
}
console.log("MUITO OBRIGADO")
console.log("Alcool: " + a)
console.log("Gasolina: " + g)
console.log("Diesel: " + d)

File: INICIANTE/JAVA/1134.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int n = sc.nextInt();
        int a = 0;
        int g = 0;
        int d = 0;

        for(;n != 4;){
            if(n == 1){
                a++;
            }
            else if(n == 2){
                g++;
            }
            else if(n == 3){
                d++;
            }
            n = sc.nextInt();
        }
        System.out.println("MUITO OBRIGADO");
        System.out.println("Alcool: " + a);
        System.out.println("Gasolina: " + g);
        System.out.println("Diesel: " + d);
        sc.close();
    }
}
